### DISCUSSION TITLE
PR 브랜치를 fork한 레포말고 woowacourse 레포에서 분기해요~

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#492] [결정 사항]
### 슬랙봇, 백엔드 테스트 액션 self-hosted
현재 우리 액션 워크플로들 중 fork한 origin의 브랜치일 경우 secret key 접속이 불가능하여 사용할 수 없는 것들이 있다.
self-hosted 일 경우 아래의 문제들이 발생한다.

- 서버 인스턴스가 종료될 경우, 모든 action이 돌지 않는다.
- 동시 실행이 되지 않으므로 github hosted runner에 비해 느리다.
- 실행이 많이 될수록 우리 인스턴스에 부하가 간다.

##  pr 분기를 fork한 레포말고 woowacourse 레포에서 분기한다.
- woowacourse의 github hosted runner에서 액션이 수행되기 때문에 위의 문제점들이 해결됨
  - 실행 환경 문제 때문에 여러번 액션파일을 수정했는데 이러지 않아도 됨 
- 초반에 브랜치를 분기처리하는 방식이 바뀌어 어색할 수 있으나 큰 어려움이 있는 작업은 아니라고 생각함

레벨4까지 내다봤을 때 장기적으로 봤을 때 지금이라도 바꾸는 게 어떨까요~?

# 추가 논의 사항
### 브랜치명을 어떻게 할 것인가?
채택 : 0안

**0. feat/이슈번호-브랜치명**
1. #이슈번호/브랜치명
2. fe/브랜치명
3. 브랜치명 안 겹치게 잘..
4. 내아이디/브랜치명

**Comment by zeus6768** [comment]
지지합니다.

**Comment by zangsu** [comment]
좋아요~

**Comment by Jaymyong66** [comment]
원격 레포지토리의 다 쓴 브랜치 정리만 잘 하면 될거 같아요~!
